Source/Engine/PackageBuilder/PatternLinker.cs

```diff
@@ -24,21 +24,17 @@ internal struct PatternReferenceInContext
         private Dictionary<string, PatternSyntax> fPatternByName;
         private List<PatternReferenceInContext> fPatternReferences;
         private PatternSyntax fCurrentPattern;
-        private List<Syntax> fExtractedPatterns;
-        private int fNextExtractedPatternNumber;
-
+        
         public PatternLinker(bool linkRequiredPackages)
```

Comment:
Can you add some simple "smoke" unit test for this class to check that after "Link" references are updated? As it's not used anymore in the search flow, I think it will be useful.